simple_gmap.info

```diff
@@ -2,4 +2,4 @@ name = Simple Google Maps
 description = Provides a Google Maps link/map formatter for simple Text fields
 backdrop = 1.x
 type = module
-package = Fields
+package = Fields
```

Comment:
same thing here, we need another linebreak.

Reply:
Done